### PR TITLE
Add square-root

### PR DESCRIPTION
**Pull request by BNAndras**
Related to #207

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -309,9 +309,9 @@
       },
       {
         "slug": "secret-handshake",
+        "practices": [],
```
Comment (ErikSchierboom) [inline config.json:312]: Was this change intentional?

**Review by ErikSchierboom — APPROVED** [review]
One tiny nit

File: config.json (hunk 1 above)
Comment (BNAndras) [reply config.json:312]: No. Thanks for catching that.
